### PR TITLE
Allow adding configuration set on emails

### PR DESCRIPTION
**Pull request by DKarim**
https://docs.aws.amazon.com/ses/latest/dg/using-configuration-sets.html

Allows defining specific events destinations and IP pools

### CODE REVIEW

File: docs/guide/src/docs/asciidoc/ses.adoc [hunk 1]
```diff
@@ -72,11 +72,21 @@ include::{root-dir}/subprojects/micronaut-amazon-awssdk-ses/src/test/groovy/com/
 <3> Define the from address
 <4> Define one or more recipients
 <5> Define HTML body (alternatively you can declare plain text body as well)
+<<<<<<< Updated upstream
```
Comment (musketyr) [inline docs/guide/src/docs/asciidoc/ses.adoc:75]: forgotten merge?

**Review by musketyr — CHANGES_REQUESTED** [review]
the code contains a lot of forgotten merge comments from unstashing

**Comment by DKarim** [comment]
> the code contains a lot of forgotten merge comments from unstashing

solved

File: subprojects/micronaut-aws-sdk-ses/src/test/groovy/com/agorapulse/micronaut/aws/ses/SendEmailSpec.groovy [hunk 2]
```diff
@@ -54,12 +54,13 @@ class SendEmailSpec extends Specification {
...
-                    filename 'test.pdf'                                                 // <9>
-                    mimeType 'application/pdf'                                          // <10>
-                    description 'An example pdf'                                        // <11>
+                configurationSetName 'configuration-set'                                    // <6>
+                tags mapOfTags                                                              // <7>
+                attachment {                                                                // <8>
+                    filepath thePath                                                        // <9>
+                    filename 'test.pdf'                                             // <10>
+                    mimeType 'application/pdf'                                          // <11>
+                    description 'An example pdf'                                        // <12>
```
Comment (musketyr) [inline subprojects/micronaut-aws-sdk-ses/src/test/groovy/com/agorapulse/micronaut/aws/ses/SendEmailSpec.groovy:63]: same as above, please, align the bullet point comments

**Review by musketyr — APPROVED** [review]
looks, good, just a few cosmetic issues for the docs

**Pull request merged**